chpl/chpl-resources/src/main/resources/errors.properties

```diff
@@ -798,6 +798,8 @@ search.developer.statuses.invalid=Could not find a developer status with value '
 search.developer.decertificationDate.invalid=Could not parse '%s' as a decertification date in the format %s.
 search.developer.missingActiveListingsOperator=Multiple active listings filters were found without a search operator (AND/OR). A search operator is required.
 search.developer.activeListingsSearchOption.invalid=No active listings search option matches '%s'. Values must be one of %s.
+search.developer.missingAttestationsOperator=Multiple attestations filters were found without a search operator (AND/OR). A search operator is required.
```

Comment:
This says a search operator is required, but the documentation of the operator says it defaults to `OR`. They both can't be right